app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] Please call the method `update_comment_counter` after save, It will update the `comments_counter` for the post when a comment is created.
  - You can consider the below code.
``` ruby
after_save :update_comment_counter